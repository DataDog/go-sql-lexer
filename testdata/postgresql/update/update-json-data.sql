UPDATE events SET data = jsonb_set(data, '{location}', '"New Location"') WHERE data->>'event_id' = '123';
