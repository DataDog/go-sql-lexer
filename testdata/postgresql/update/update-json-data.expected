[
    {
        "expected": "UPDATE events SET data = jsonb_set ( data, ?, ? ) WHERE data ->> ? = ?",
        "statement_metadata": {
            "size": 12,
            "tables": [
                "events"
            ],
            "commands": [
                "UPDATE"
            ],
            "comments": [],
            "procedures": []
        }
    },
    {
        "expected": "UPDATE events SET data = jsonb_set(data, ?, ?) WHERE data ->> ? = ?",
        "normalizer_config": {
            "remove_space_between_parentheses": true
        }
    }
]